src/FieldEncryptMapPluginManager.php

```diff
@@ -0,0 +1,30 @@
+<?php
+
+/**
+ * @file
+ * Contains \Drupal\field_encrypt\FieldEncryptMapPluginManager.
+ */
+
+namespace Drupal\field_encrypt;
+
+use Drupal\Core\Cache\CacheBackendInterface;
+use Drupal\Core\Extension\ModuleHandlerInterface;
+use Drupal\Core\Plugin\DefaultPluginManager;
+
+/**
+ * Class FieldEncryptMapPluginManager
+ * @package Drupal\field_encrypt
+ */
+class FieldEncryptMapPluginManager extends DefaultPluginManager {
+
+  /**
+   * This is copied mostly from the DefaultPluginManager.
```

Comment:
This will barf in PHPCS..

 /**
   * Constructs a new KeyProviderPluginManager.
   *
   * @param \Traversable $namespaces
   *   An object that implements \Traversable which contains the root paths
   *   keyed by the corresponding namespace to look for plugin implementations.
   * @param \Drupal\Core\Cache\CacheBackendInterface $cache_backend
   *   Cache backend instance to use.
   * @param \Drupal\Core\Extension\ModuleHandlerInterface $module_handler
   *   The module handler.
   */